marlowe-playground-client/static/css/panels.scss

```diff
@@ -263,29 +262,37 @@ button.minus-btn:hover {
   outline: none;
 }
 
-.metadata-field-group {
-  margin: 0.5rem;
-}
-
-.metadata-field-group h6 {
-  margin-top: 1rem;
-  margin-bottom: 0.5rem;
+.metadata-group-title h6 {
   font-weight: bold;
-}
-
-.metadata-field-group li {
-  margin-left: 1rem;
-}
-
-.metadata-field-group li::marker {
-  display: none;
-  content: " ";
+  margin: 0.5rem 0;
 }
 
 .metadata-error {
   color: var(--error-color);
 }
 
 .metadata-not-used {
-  margin-left: 1rem;
-}
\ No newline at end of file
+  row-gap: 1rem;
+}
+
+.metadata-form {
+  display: grid;
+  grid-template-columns: auto 1fr auto auto;
```

Comment:
Not sure what is the 4th grid column. I gather than first is the label, second is the input, third is the button, and the 4th?

Reply:
That one is for the warning when you have a field that is not in the contract

Reply:
![Screenshot_20210315_142757](https://user-images.githubusercontent.com/638102/111169436-daf6d780-859a-11eb-9fa3-41a0bf280df6.png)